src/plugins/git/createGraph.test.js

```diff
@@ -143,11 +146,67 @@ describe("createGraph", () => {
             direction: "OUT",
           })
         ).toHaveLength(1);
-        expect(graph.neighborhood(entryAddress)).toHaveLength(2);
+        const becomesCount = graph.neighborhood(entryAddress, {
+          edgeType: BECOMES_EDGE_TYPE,
+        }).length;
+        ["OUT", "IN"].forEach((direction) => {
+          expect(
+            graph.neighborhood(entryAddress, {
+              edgeType: BECOMES_EDGE_TYPE,
+              direction,
+            }).length
+          ).toBeLessThanOrEqual(1);
+        });
+        expect(graph.neighborhood(entryAddress)).toHaveLength(becomesCount + 2);
       });
     });
   });
 
+  it('has "becomes" edges with valid paths', () => {
+    const data = makeData();
+    const graph = createGraph(data);
+    const becomings: $ReadOnlyArray<Edge<BecomesEdgePayload>> = graph
+      .edges({type: BECOMES_EDGE_TYPE})
+      .map((edge) => ((edge: Edge<any>): Edge<BecomesEdgePayload>));
+    expect(becomings).not.toHaveLength(0);
+    becomings.forEach((edge) => {
+      expect(edge.dst).not.toEqual(edge.src);
+      const {payload: {childCommit, parentCommit, path}} = edge;
+      expect(path).not.toHaveLength(0);
+      expect(data.commits[childCommit].parentHashes).toEqual(
+        expect.arrayContaining([parentCommit])
+      );
+      function expectedTreeEntryAddress(commit: Hash): Address {
```

Comment:
Let's test that edge.src and edge.dst are valid addresses to nodes in the graph, and those nodes have expected properties.

Reply:
Sure; done.